core/server/helpers/excerpt.js

```diff
@@ -12,6 +12,24 @@ var hbs = require('express-hbs'),
 function excerpt(options) {
     var truncateOptions = (options || {}).hash || {};
```

Comment:
All the try/catch blocks are unneeded because if the append/roundSentence/stripTags properties are passed to the helper they will be picked up by this line.